contracts/Aavegotchi/libraries/LibWhitelist.sol

```diff
@@ -65,4 +67,29 @@ library LibWhitelist {
             _removeAddressFromWhitelist(_whitelistId, _whitelistAddresses[i]);
         }
     }
+
+    function setWhitelistAccessRight(
+        uint32 _whitelistId,
+        uint256 _actionRight,
+        uint256 _accessRight
+    ) internal {
+        require(_isAccessRightValid(_actionRight, _accessRight), "LibWhitelist: Invalid Rights");
+        AppStorage storage s = LibAppStorage.diamondStorage();
+        s.whitelistAccessRights[_whitelistId][_actionRight] = _accessRight;
+        emit WhitelistAccessRightSet(_whitelistId, _actionRight, _accessRight);
+    }
+
+    function _isAccessRightValid(uint256 _actionRight, uint256 _accessRight) internal pure returns (bool) {
+        // This action right limits borrowers in a whitelist to a number of borrowed gotchis. 0 is unlimited

```

Comment:
@pakim249CAL This function could be simplied. Like this.
`return _actionRight == 0;`

Reply:
I wanted to write it this way to make clear that this is where devs can add more access right logic in the future.